Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM adoptopenjdk/openjdk14:jre-14.0.2_12
```

Comment:
This will need the licence header for maven builds to work, unless you exclude it in the licence check config.

Reply:
the Dockerfile and Makefile have been excluded in the parent pom